src/transformers/trainer.py

```diff
@@ -32,6 +32,7 @@
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Union
 
 from tqdm.auto import tqdm
+from torch_ort import ORTModule
```

Comment:
This will make torch_ort a dependency for using transformers. I'm not sure we can upstream this dependency. Consider qualifying the import with ort flag.

Reply:
Makes sense.